hieradata_aws/class/jenkins.yaml

```diff
@@ -54,10 +54,18 @@ govuk_jenkins::config::banner_string: 'AWS Deploy'
 
 govuk_jenkins::job_builder::jobs:
   - govuk_jenkins::job::deploy_app
+  - govuk_jenkins::job::deploy_emergency_banner
   - govuk_jenkins::job::deploy_puppet
+  - govuk_jenkins::job::deploy_router_data
```

Comment:
Do we need to change the PLEK values of the job to work on AWS?

Reply:
This job is rarely used, so I'm going to remove for now.